### PR TITLE
Removing STL headers from filameshio

### PR DESCRIPTION
**Pull request by nitronoid**
Removing STL headers from the filamesh reader. Not sure that the jsbindings were amended correctly. Also still has `<functional>` from the `CString` header using it for `std::hash` specialisation, maybe that could be moved to another header to be included only by internal source files, as `std::hash` should only be used by STL headers anyway? (I think).

### CODE REVIEW

**Comment by nitronoid** [comment]
I signed it!

**Comment by prideout** [comment]
This seems reasonable and I think jsbindings look good too, but you can review our `CODE_STYLE.md` document and fix up the formatting?

**Comment by nitronoid** [comment]
@prideout Sorry I forgot about that, hopefully it's in line with the guide now.

File: libs/filameshio/include/filameshio/MeshReader.h [hunk 1]
```diff
@@ -47,31 +47,34 @@ class MeshReader {
     class MaterialRegistry
     {
```
Comment (prideout) [inline libs/filameshio/include/filameshio/MeshReader.h]: move brace up

Comment (nitronoid) [reply libs/filameshio/include/filameshio/MeshReader.h]: Oops, thank you

**Pull request merged**